src/overlays/actors/ovl_En_Mnk/z_en_mnk.h

```diff
@@ -7,10 +7,57 @@
 struct EnMnk;
 
 typedef void (*EnMnkActionFunc)(struct EnMnk*, PlayState*);
+typedef void (*EnMnkFunc)(PlayState*, struct EnMnk*);
+
+#define MONKEY_GET_TYPE(this) ((((Actor*)(this))->params & 0x780) >> 7)
+#define MONKEY_GET_SWITCHFLAG(this) (((Actor*)(this))->params & 0x7F)
+#define MONKEY_GET_PATH_INDEX(this) ((((Actor*)(this))->params & 0xF800) >> 11)
```

Comment:
```suggestion
#define MONKEY_GET_TYPE(thisx) (((thisx)->params & 0x780) >> 7)
#define MONKEY_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
#define MONKEY_GET_PATH_INDEX(thisx) (((thisx)->params & 0xF800) >> 11)
```
This macros should take the base actor pointer directly, if you passed in `this` directly you would need to change it to pass in `&this->actor`